src/helpers/uriHandlerController.ts

```diff
@@ -33,8 +34,9 @@ export class UriHandlerController implements UriHandler {
             txHash: searchParams.get('txHash') ?? undefined,
             workingDirectory: searchParams.get('workingDirectory') ?? undefined,
             providerUrl: searchParams.get('providerUrl') ?? undefined,
+            network: searchParams.get('network') ?? undefined,
             disableFetchExternal: !!searchParams.get('disableFetchExternal'),
-          };
+          } as DebuggerTypes.DebugArgs;
```

Comment:
Do we need this type assertion here?

Reply:
I put only to make explicit the type, but no problem in remove that

Reply:
Got it. I don't mind having explicit types. The issue with `as` type assertion is that might silent future errors. I would use something like

```ts
const args: DebuggerTypes.DebugArgs = {
  // ...
};
```